js-plugins/visualizer/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@farmfe/js-plugin-record-viewer",
-  "version": "2.2.0",
+  "name": "@farmfe/visualizer",
```

Comment:
rename it to `@farmfe/js-plugin-visualizer`

Reply:
done